### PR TITLE
Rename generate_analysis_graphs -> automated_analysis

### PR DESCRIPTION
**Pull request by as2388**
I had been reluctant to make non-essential changes on this project, but as we start to add mapping to the automated analysis stage the naming has become actively confusing, so requesting to change now.

### CODE REVIEW

File: docker-run-automated-analysis.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ if [[ "$PROFILE_CPU" = true ]]; then
     PROFILE_CPU_CMD="-m pyinstrument -o /data/cpu.prof --renderer html --"
     SYS_PTRACE_CAPABILITY="--cap-add SYS_PTRACE"
 fi
-CMD="pipenv run python -u $PROFILE_CPU_CMD generate_analysis_graphs.py \
+CMD="pipenv run python -u $PROFILE_CPU_CMD automated_analysis.py \
```
Comment (IsaackMwenda) [inline docker-run-automated-analysis.sh:47]: change the image name too.

File: run_scripts/5_automated_analysis.sh [hunk 2]
```diff
@@ -31,7 +31,7 @@ DATA_ROOT=$4
 mkdir -p "$DATA_ROOT/Outputs"
 
 cd ..
-./docker-run-generate-analysis-graphs.sh ${CPU_PROFILE_ARG} \
+./docker-run-automated-analysis.sh ${CPU_PROFILE_ARG} \
```
Comment (IsaackMwenda) [inline run_scripts/5_automated_analysis.sh:34]: edit the usage statement too

**Review by IsaackMwenda — APPROVED** [review]

File: docker-run-automated-analysis.sh (hunk 1 above)
Comment (as2388) [reply docker-run-automated-analysis.sh:47]: thanks, fixed

File: run_scripts/5_automated_analysis.sh (hunk 2 above)
Comment (as2388) [reply run_scripts/5_automated_analysis.sh:34]: fixed

**Pull request merged**